zeppelin-web/src/app/interpreter/interpreter.css

```diff
@@ -109,3 +109,696 @@
   color: #000;
   opacity: .5;
 }
+
+/**
+ * github flavored markdown style
+ */
```

Comment:
@Leemoonsoo 

I'v moved license file and description to `LICENSE` and `licenses/` from `zeppelin-distribution/src/bin_license/` 
Also, I extracted the markdown css code from `interpreter.css` to clarify its license. (297733f)